go.mod

```diff
@@ -15,5 +15,6 @@ require (
 	k8s.io/api v0.0.0-20190409021203-6e4e0e4f393b
 	k8s.io/apimachinery v0.0.0-20190404173353-6a84e37a896d
 	k8s.io/client-go v11.0.1-0.20190409021438-1a26190bd76a+incompatible
-	k8s.io/klog v0.3.0
+	k8s.io/code-generator v0.17.1
```

Comment:
I tried this approach, but it looks like you're getting the same result that I did: which is that it forces you to use the latest version of the `code-generator` (which produced client code that was incompatible with the versions of `k8s.io/api` and `k8s.io/apimachinery` we're using).

Reply:
hrmmm i locked it.. sec